.env.sample

```diff
@@ -13,9 +13,7 @@ WEBHOOK_LISTENER_PORT=8080
 # this is where workflows are defined, from which they will be imported into the database
 WORKFLOWS_DEF_TOML_FILE=./resources/workflows.toml
 
-#
 # Matrix
-#
-#MATRIX_SERVER_NAME=matrix.test:443
-#MATRIX_USERNAME=morpheus
-#MATRIX_PASSWORD=redpill
+MATRIX_SERVER_NAME=localhost:8008
```

Comment:
The reason why I had these commented out is to support "quick start demo" of workflow which requires default state of zero matrix involvement.

Reply:
And since our production has a .env file of its own and doesn't use this sample file to generate one (correct me if I am wrong?) then why do we need to uncomment these in this PR?

I am down for this change though, but want to address the surrounding bits.

Reply:
Indeed, not the scope of this PR, will revert this commit.